### DISCUSSION TITLE
У mintmanga новый сайт

### DISCUSSION THREAD

**Discussion by HaruKunOFF** [lirix360/ReadmangaGrabber#95] [Ideas]
Можете пожалуйста добавить новый сайт у mintmanga, а именно 24.mintmanga.one

**Comment by angry-master-yoda** [comment]
Присоединяюсь.
С этого сайта ничего не качает. И в избранное ссылки на него не добавляются.

**Reply by ghost** [reply]
Попробуйте другую цифру (поддомен)

**Comment by ghost** [comment]
> Можете пожалуйста добавить новый сайт у mintmanga, а именно 24.mintmanga.one

Это всего лишь поддомен. Сегодня он работает - завтра нет. Сайт один и тот же. Причем на оригинальном mintmanga.one размещен не сайт, а заглушка. То что вы пишите - не новый сайт. Просто считайте зеркало для отвлечения. Сейчас 24 и 3 не работают, работает например 1 и 2. а то есть 1.mintmanga,one и 2.mintmanga,one
